### PR TITLE
Hbs strategy

### PR DESCRIPTION
**Pull request by betocantu93**
This PR depends on #185, its the actual hbs strategy implementation from #166

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -108,7 +108,7 @@
   "husky": {
     "hooks": {
       "pre-commit": "lint-staged",
-      "pre-push": "yarn lint:js && yarn test:node"
+      "pre-push": "yarn lint:js && yarn test:ember"
```
Comment (jherdman) [inline package.json:111]: Would you mind also adding the node tests to this too?

Comment (betocantu93) [reply package.json:111]: sure, done! 👍🏼

**Review by jherdman — APPROVED** [review]
LGTM, though one last request: can you rebase away all of the merge commits? In fact, just squashing this down to a single commit would be ideal.

Lastly, I'd love to hear from @ivanvotti re dropping a bunch of the old Ember's from the build pipeline.

**Comment by betocantu93** [comment]
@jherdman nooot sure if I did it correctly lol, never done these kind of git operations manually! it looks fine I think

**Comment by jherdman** [comment]
OK! We're down to the final steps! One last rebase and we're good to go!

**Comment by betocantu93** [comment]
@jherdman ready ! 👍🏼 👍🏼

**Review by jherdman — APPROVED** [review]

**Pull request merged**